src/components/form-elements/formik/README.md

```diff
@@ -60,6 +61,24 @@ const SelectField = require('box-ui-elements/es/components/select-field').Select
                     placeholder="Text Area Field"
                     component={TextArea}
                 />
+                <b>Non-RadioGroup RadioButtons sharing the same name</b>
+                <br />
+                <br />
+                <Field
+                    isSelected={false}
+                    label="Radio Button Field 1"
+                    name="radiobutton"
```

Comment:
This example is a bit confusing. Radio buttons with the same name are meant to be part of a group.

Reply:
Yeah radio will still be used as a group of radios (hence same name) just not inside our `<RadioGroup>` component since that does not allow arbitrary components.